config/crd/bases/infinispan.org_infinispans.yaml

```diff
@@ -921,13 +921,18 @@ spec:
                               port:
                                 format: int32
                                 type: integer
+                              routerHostName:
```

Comment:
Here is inconsistency between field name and the description. While the field is `routerHostName` the description mentions `RouteHostName` (route vs router), probably should be `route`